cmd/nginx/main.go

```diff
@@ -47,15 +47,6 @@ import (
 	"k8s.io/ingress-nginx/version"
 )
 
-const (
-	// High enough QPS to fit all expected use cases. QPS=0 is not set here, because
-	// client code is overriding it.
-	defaultQPS = 1e6
-	// High enough Burst to fit all expected use cases. Burst=0 is not set here, because
-	// client code is overriding it.
-	defaultBurst = 1e6
```

Comment:
what are the default kubeclient configurations?

Reply:
https://github.com/kubernetes/client-go/blob/2a3e58aa702638e5d85f3c24028b3a2109ef1d7f/rest/config.go#L110
For context, kubelet uses the default values